maquetado/scss/tech.scss

```diff
@@ -2,6 +2,15 @@
 @import 'fonts';
 @import 'sizes';
 
+.main-container {
+    padding: 50px;
+}
+
+.tc-container-fe, .tc-container-be {
```

Comment:
I think that you can use only one class, `.tc-container` instead of two with the same CSS rules.